packages/web3-eth/test/fixtures/rpc_methods_with_params.ts

```diff
@@ -0,0 +1,197 @@
+import { TransactionCall, TransactionWithSender } from 'web3-common';
+import {
+	Address,
+	BlockNumberOrTag,
+	BlockTags,
+	Filter,
+	HexString32Bytes,
+	HexStringBytes,
+	Uint,
+	Uint256,
+} from 'web3-utils';
+
+// TODO Not sure if this is okay to do
+import { isFilterObjectValidData } from 'web3-utils/test/fixtures/validation';
```

Comment:
@nazarhussain Is this going to cause problems? Can't import it regularly since it's not being exported in `index.ts`

Reply:
Yes we should not import anything from `test` directory. If that's really a reusable piece of data/code then it must be kept in `src`. One can import from the source directory with a particular path.

Reply:
If I understand this correctly, doesn't this data need to be included when the `web3-utils` package is built? If it's not, then the `web3-eth` package is no longer standalone (would require pulling the un-built `web3-utils`), because it would import files not included in the published version of `web3-utils`

I don't think we should include test data in the published version of `web3-utils`, so maybe we just duplicate `isFilterObjectValidData` in `web3-eth`?